sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1037,12 +1037,12 @@ object RapidsConf {
     .createWithDefault(true)
 
   val ENABLE_REGEXP = conf("spark.rapids.sql.regexp.enabled")
-    .doc("Specifies whether regular expressions should be evaluated on GPU. Complex expressions " +
-      "can cause out of memory issues so this is disabled by default. Setting this config to " +
-      "true will make supported regular expressions run on the GPU. See the compatibility " +
-      "guide for more information about which regular expressions are supported on the GPU.")
+    .doc("Specifies whether supported regular expressions should be evaluated on GPU. There " +
+      "are some known edge cases that can produce incorrect results and these are documented in " +
```

Comment:
should this say that known edge cases are executed on CPU, at least this is the impression I have from reading compatibility guide.